packages/actions/types/Actions.d.ts

```diff
@@ -495,6 +552,11 @@ declare module 'grafx-studio-actions' {
              */
             getBooleanValue(name: string | Variable): boolean;
 
+            /**
+             * Gets the number value of a variable
+             */
+            getNumberValue(name: string | Variable): number;
```

Comment:
I'm a bit lost with all the helpers, don't we want a `getNumber`(variable) here as well?

Reply:
In this file only the typings are defined, these should match with the `studio` object being exposed by the `engine`. 

> don't we want a getNumber(variable) here as well?

I don't think so, `studio.variables.byName("myNumberVar")` already returns a `NumberVariable` instance. But it is typed as the generic `Variable`. 

To help with this an actual helper function is defined here:

https://github.com/chili-publish/studio-sdk/pull/462/files#diff-2e9f181658fcc70ebad720fcb2c98c9a0a64a357b9e609c73368f64a6de52a16R44